dashboard/app/models/concerns/unit_concerns/text_to_speech.rb

```diff
@@ -0,0 +1,23 @@
+module UnitConcerns
```

Comment:
This needed to be renamed, as the `Unit` module defined by the previous implementation conflicted with the `Unit` class that depends on it. To be honest, I'm not entirely sure how the autoloader resolves that conflict; but because we can no longer rely entirely on the autoloader for this code path, we need to navigate around it ourselves.

If anyone has any better ideas for how we'd prefer to organize this particular part of our codebase, I'd love to hear them!